src/scm/github.js

```diff
@@ -0,0 +1,433 @@
+'use strict';
+
+class Github {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/76f8108a25ffe1858d5d27c3d7dabf83cc8b0291/src/scm/github.js#L3"></span>‘Github’ is defined but never used.</td>
    </tr>
  </tbody>
</table>